src/backends/kafka/config.rs

```diff
@@ -0,0 +1,106 @@
+use rdkafka::config::ClientConfig as RdKafkaConfig;
+use std::collections::HashMap;
+
+const DEFAULT_QUEUED_MAX_MESSAGE_KBYTES: u32 = 50_000;
+const DEFAULT_QUEUED_MIN_MESSAGES: u32 = 10_000;
+
+#[derive(Debug, Clone)]
+pub struct KafkaConfig {
+    config_map: HashMap<String, String>,
+}
+
+impl KafkaConfig {
+    pub fn new_config(bootstrap_servers: Vec<String>) -> Self {
+        let mut config = HashMap::new();
+        config.insert("bootstrap.servers".to_string(), bootstrap_servers.join(","));
+
+        Self { config_map: config }
+    }
```

Comment:
Do we need this one ?

Reply:
I think so. We have it in the Python library. We'll probably use it heavily in tests with the admin client for topic management, etc